py/CHANGES

```diff
@@ -1,3 +1,6 @@
+Selenium (TBA version)
```

Comment:
minor: I think we are gonna go with a `4.6.1` here, can you update that? could I also ask you to add a line about the `__del__` changes too, would be greatly appreciated